docs/parcel-api.md

```diff
@@ -0,0 +1,50 @@
+# Parcel API
+Most parcel methods will be called on the parcel itself, with the excpetion being around mounting.
+
+## mounting
+Both mount methods return a [parcel object](/docs/parcel-api.md#parcel-object). The parcel object contains all additional exposed methods.
+### mountParcel
+`application.mountParcel(parcelConfig)`. Each application is provided a mountParcel function.
+The main advantage to using an applications `mountParcel` function is that parcels mounted via an 
+applications `mountParcel` will be automatically unmounted when the application is unmounted.
+
+### mountRootParcel
+The [mountRootParcel](/docs/single-spa-api.md#mountrootparcel) method will mount the parcel but `unmount` must be called manually.
+
+## Parcel Object
+The parcel object is the external representation of a single-spa parcel. It contains the following functions and methods:
```

Comment:
nit:

External to what? I know what you're referring to here -- our implementation of parcels has an internal representation and an external representation. But from the user's perspective, isn't there only just the one representation?